.ci/travis/travis.sh

```diff
@@ -59,7 +59,7 @@ sonarqube)
 release-dry-run)
   if [ $(git log -1 | grep -E "\[maven-release-plugin\] prepare release" | cat | wc -l) -lt 1 ]; then
     mvn -e release:prepare -DdryRun=true --batch-mode -Darguments='-DskipTests -DskipITs \
-      -Dcobertura.skip=true -Dpmd.skip=true -Dfindbugs.skip=true  -Dxml.skip=true \
+      -Dcobertura.skip=true -Djacoco.skip=true -Dpmd.skip=true -Dfindbugs.skip=true  -Dxml.skip=true \
```

Comment:
jacoco is defined in special profile, other builds should not be affected.
please remove.

Reply:
I'm looking a bit forward. Reverted for now.